tests/test_dataset.py

```diff
@@ -487,3 +490,65 @@ def test_sanitized_dimensions(spark_context):
 
     summaries = dataset.summaries(spark_context)
     assert len(summaries) == 2
+
+
+@pytest.fixture
+def setupDF():
+    bucket_name = 'test_bucket'
+    store = InMemoryStore(bucket_name)
+    store.store['dir1/subdir1/key1'] = json.dumps({'foo': 1})
+    store.store['dir2/subdir2/key2'] = json.dumps({'foo': 2})
+    dataset = Dataset(bucket_name, ['dim1', 'dim2'], store=store)
+    return dataset
+
+
+@pytest.mark.slow
```

Comment:
I would drop this for consistency. None of the other tests are using this slow test marker.